Autor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+class Autor {
```

Comment:
```suggestion
<?php

class Autor {
```
pon un espacio entre 'php' y 'class Autor'